src/classes/Foo5.cls

```diff
@@ -0,0 +1,15 @@
+public class Foo5 {
+    void foo() {
+        
+        
+        
+        String myString = 'MY_HARDCODED_STRING';
+        Blob hardCodedIV = Blob.valueOf('Hardcoded IV 123');
+        Blob myKey = Blob.valueOf(myString);
+        Blob data = Blob.valueOf('Data to be encrypted');
+        Blob encrypted = Crypto.encrypt('AES128', myKey, hardCodedIV, data);
+        
+        
+        List<Order__c> orders = [SELECT Purchased__c,Product_Category_1__c,Product_Category_Units_1__c,Product_Category_2__c,Product_Category_Units_2__c FROM Order__c];
```

Comment:
The result of this query is not bound. Please add either a `LIMIT` or a `WHERE` statement to the query to limit the number of results.